common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -59,6 +60,5 @@
 </form>
 
 % if msg:
-<span class="message">${msg|n}</span>
+<span class="message">${HTML(msg)}</span>
```

Comment:
> 1. HTML-escape the plain text before it gets here so it too is HTML by the time you get here, or

@robrap According to this ^^,  the plain text will be HTML escaped with the current approach as well as HTML along with HTML content. If yes then why we need to HTML escape the plain text in python explicitly?